recipe/meta.yaml

```diff
@@ -22,6 +22,8 @@ requirements:
     - python >=3.6
   run:
     - python >=3.6
+    - numpy
+    - numba
```

Comment:
numba should be optional.

cc @ml31415

Reply:
I added it since it now appears in the `install_requires` section of the `setup.py`:
https://github.com/ml31415/numpy-groupies/blob/master/setup.py#L73

Reply:
Oh thanks! I missed that.